layers/+lang/bibtex/README.org

```diff
@@ -82,3 +117,24 @@ key binding added:
 | Key binding | Description     |
 |-------------+-----------------|
 | ~SPC m i c~ | Insert citation |
+
+*EBIB*
+
+*Index buffer*
+| Key Binding     | Description                                           |
+|-----------------+-------------------------------------------------------|
+| ~J/K~           | scroll page down/up                                   |
+| ~SPC m j~       | jump to entry (searches only author and title fields) |
```

Comment:
normally "jumping"-like functionalities are under "g" prefix.
so I suggest `SPC m g g`.

Reply:
No need to prefix with `SPC m` I propose to add `g j` for jumping to an item and leave this keybinding here also (or you think I should remove the current one?)